test/shlibloadtest.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
done.

I checked the Copyright headers in that directory and I wanted to ask if that one is correct:
x509_dup_cert_test.c: * Copyright (c) 2017 Oracle and/or its affiliates.  All rights reserved.